workflows/datalogger/generic/find_last_known_machine_positions/find_last_known_machine_positions.py

```diff
@@ -50,6 +50,7 @@ def path_up_to_last(a_last, a_inclusive=True, a_path=os.path.dirname(os.path.rea
 
 # get the datalogger data
 response = session.get(dba_url, headers=headers)
+logging.debug(response.text)
```

Comment:
did you mean to leave this in?

Reply:
Yes, thought it would be useful in debug to access the response.